spec/jobs/subscription_confirm_job_spec.rb

```diff
@@ -168,6 +176,7 @@
           end
 
           it "sends only a subscription confirm email, no regular confirmation emails" do
+            ActionMailer::Base.perform_deliveries = true
```

Comment:
We must restore its value afterward to avoid messing the global state. You can use `performing_deliveries` for that as we did in spec/features/admin/users_spec.rb

Reply:
Thanks, @sauloperez. Yes, I agree. I've fixed this here: https://github.com/openfoodfoundation/openfoodnetwork/pull/3650/commits/f7b47fb622ceaafed01d0fab775f8cd001352651

Reply:
I spoke too soon. This is the correct commit: https://github.com/openfoodfoundation/openfoodnetwork/pull/3650/commits/f0746eb64387ba17cc655f79dd57bf5efd554a10 Build is green again.